modules/lvgl/Kconfig.input

```
@@ -4,11 +4,33 @@
 
 menu "Input device settings"
 
+DT_CHOSEN_ZEPHYR_TOUCH := zephyr,touch
```

```suggestion
DT_CHOSEN_Z_TOUCH := zephyr,touch
```
Seems to be the predominant naming throughout the tree, non-blocking.
https://github.com/search?q=repo%3Azephyrproject-rtos%2Fzephyr+DT_CHOSEN_Z_&type=code